### PR TITLE
Feature/pptp 1423

### PR DESCRIPTION
**Pull request by kiranprakashdigital**
### Description of Work carried through

Brief description of what/why/how.

#### Check list 
 - [x] `./precheck` was executed (Integration/Component/Unit tests)
 - [x] `sbt scalafmt test:scalafmt` was executed
 - [x] Required Environment Config has been amended/added

### CODE REVIEW

**Review by paulblakehmrc — APPROVED** [review]
Good effort this - LGTM.

File: app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/partner/PartnerContactAddressControllerBase.scala [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    }
+
+  protected def doAlfCallback(
+    partnerId: Option[String],
+    id: Option[String],
+    successfulRedirect: Call
+  ): Action[AnyContent] =
+    (authenticate andThen journeyAction).async { implicit request =>
+      addressLookupFrontendConnector.getAddress(
+        id.getOrElse(throw new MissingAddressIdException)
```
[the remark this answers is not in the document]
Comment (kiranprakashdigital) [reply app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/partner/PartnerContactAddressControllerBase.scala:67]: This `id` is returned from AddresslookupFrontend. So it is made optional

File: test/uk/gov/hmrc/plasticpackagingtax/registration/views/amendment/partner/AmendAddPartnerContactCheckAnswersViewSpec.scala [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+  routes => partnerRoutes
+}
+import uk.gov.hmrc.plasticpackagingtax.registration.forms.contact.Address
+import uk.gov.hmrc.plasticpackagingtax.registration.forms.organisation.PartnerTypeEnum
+import uk.gov.hmrc.plasticpackagingtax.registration.models.genericregistration.Partner
+import uk.gov.hmrc.plasticpackagingtax.registration.services.CountryService
+import uk.gov.hmrc.plasticpackagingtax.registration.views.html.amendment.partner.amend_add_partner_contact_check_answers_page
+import uk.gov.hmrc.plasticpackagingtax.registration.views.tags.ViewTest
+
+@ViewTest
```
[the remark this answers is not in the document]
Comment (kiranprakashdigital) [reply test/uk/gov/hmrc/plasticpackagingtax/registration/views/amendment/partner/AmendAddPartnerContactCheckAnswersViewSpec.scala:33]: It is annotation to indicate it is View test similar to unit test.

[the remark this answers is not in the document]
Comment (liampan) [reply test/uk/gov/hmrc/plasticpackagingtax/registration/views/amendment/partner/AmendAddPartnerContactCheckAnswersViewSpec.scala:33]: but does it do anything? I didnt add it to any view spec's is all.

**Pull request merged**